README.md

```diff
@@ -12,9 +12,6 @@ This repository contains the following Rust crates:
 
 * [smt_trie](./smt_trie/README.md): A collection of types and functions to work with Polygon Hermez Sparse Merkle Trees (SMT).
 
-* [trace_decoder](./trace_decoder/Cargo.toml): Flexible protocol designed to process Ethereum clients trace payloads into an IR format that can be
```

Comment:
There are still other mentions of the `trace_decoder` in the README with erroneous relative path following the move